docs/timeseries/lombscarglemb.rst

```diff
@@ -0,0 +1,413 @@
+an associated band label column, these columns may be passed directly to
+:class:`~astropy.timeseries.LombScargleMultiband` as 1-d arrays.
+
+Example
+-------
+
+.. EXAMPLE START: Loading from a :class:`~astropy.timeseries.TimeSeries` object
+
+Consider the following generator code for a
+:class:`~astropy.timeseries.TimeSeries` object. Where timeseries data is
+populated for three photometric bands (g,r,i).
+
+>>> from astropy.timeseries import LombScargleMultiband, TimeSeries
+>>> import numpy as np
+>>> import astropy.units as u
+
+>>> rng = np.random.default_rng(1)
+>>> deltas = 240 * rng.random(180)
+>>> ts1 = TimeSeries(time_start="2011-01-01T00:00:00",
+...                  time_delta=deltas*u.minute)
+
+>>> # g band fluxes
+>>> ts1['g_flux'] = [np.nan] * 180 * u.mJy
+>>> ts1['g_err'] = [np.nan] * 180 * u.mJy
+>>> y_g = 3 + 2 * np.sin(10 * np.pi * ts1['time'].mjd[0:60])
+>>> dy_g = 0.01 * (0.5 + rng.random(60)) # uncertainties
+>>> ts1['g_flux'].value[0:60] = y_g
+>>> ts1['g_err'].value[0:60] = dy_g
+>>> # r band fluxes
+>>> ts1['r_flux'] = [np.nan] * 180 * u.mJy
+>>> ts1['r_err'] = [np.nan] * 180 * u.mJy
+>>> y_r = 3 + 2 * np.sin(10 * np.pi * ts1['time'].mjd[60:120])
+>>> dy_r = 0.01 * (0.5 + rng.random(60)) # uncertainties
+>>> y_r += dy_r * rng.standard_normal(60)
+>>> ts1['r_flux'].value[60:120] = y_r
+>>> ts1['r_err'].value[60:120] = dy_r
+>>> # i band fluxes
+>>> ts1['i_flux'] = [np.nan] * 180 * u.mJy
+>>> ts1['i_err'] = [np.nan] * 180 * u.mJy
+>>> y_i = 3 + 2 * np.sin(10 * np.pi * ts1['time'].mjd[120:])
+>>> dy_i = 0.01 * (0.5 + rng.random(60)) # uncertainties
+>>> y_i += dy_i * rng.standard_normal(60)
+>>> ts1['i_flux'].value[120:] = y_i
+>>> ts1['i_err'].value[120:] = dy_i
+
+Our timeseries data is set up to be asynchronous, where a given timestamp
+corresponds to a measurement in a single band. However, if your data instead
+has one timestamp per multiple band measurements, or a mixture,
+:class:`~astropy.timeseries.LombScargleMultiband` will still be able to
+operate on it.
+
+To operate on the example :class:`~astropy.timeseries.TimeSeries`,
+:class:`~astropy.timeseries.LombScargleMultiband` has a loader function, as
+follows:
+
+>>> ls = LombScargleMultiband.from_timeseries(ts1, signal_column_name=['g_flux', 'r_flux', 'i_flux'],
+...                                           uncertainty=['g_err', 'r_err', 'i_err'],
+...                                           band_labels=['g', 'r', 'i'])
+
+``signal_column_name`` requires a list of columns that correspond to the flux
+or magnitude measurements in each band. ``uncertainty`` and ``band_labels`` are
+optional, but if specified must be lists of equal size to
+``signal_column_name``. ``uncertainty`` specifies the columns containing the
+associated errors per band, while ``band_labels`` provides the labels to use
+for each photometric band. From here,
+:class:`~astropy.timeseries.LombScargleMultiband` can be worked with as normal.
+
+>>> frequency,power = ls.autopower()
```

Comment:
Is this an example of how it can be used as normal? If so, prefix with 'for example:'